apps/chat/src/store/prompts/prompts.epics.ts

```diff
@@ -586,6 +586,10 @@ const uploadPromptsWithFoldersRecursiveEpic: AppEpic = (action$, state$) =>
               FeatureType.Prompt,
             );
 
+            const rootPromptId = prompts.toSorted(
+              (a, b) => a.folderId.length - b.folderId.length,
+            )[0].id;
```

Comment:
```suggestion
            const rootPromptId = conversations.filter(p => isRootPromptId(p.id))[0].id;
```

Reply:
-//-
So, I've now renamed it as topLevelPromptId.